packages/styles/scss/components/buttongroup/_buttongroup.scss

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/imports';
+
+@mixin buttongroup {
+  $btn-min-width: 26;
+
+  .#{$prefix}--buttongroup {
+    @include carbon--make-row;
+
+    &__ctas {
+      @include carbon--make-col-ready;
```

Comment:
Do we need to include layout SCSS mixins like this and row above if it's just a full width? If we remove these adopters can put it wherever they need it in their page's layout.

Reply:
Took a closer look, makes sense. I removed the outer layer and just made it 100% width.
cc: @annawen1 since this would affect your pattern if you're bringing this back in.